Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
+
+# Install Python and necessary dependencies
+RUN apt-get update && \
+    apt-get install -y python3 python3-pip git && \
+    rm -rf /var/lib/apt/lists/* && \
+    ln -s /usr/bin/python3 /usr/bin/python && \
+    pip3 install setuptools
+
+ENV PYTHONPATH=/scrutinycspm:/scrutinycspm/src:/scrutinycspm/cli
```

Comment:
This shouldn't be needed if we could use setup tools or something that help you set up a python project.

Reply:
When moving to a docker container, pathing is broken.  This was a simple fix.